packages/compass/src/main/window-manager.ts

```diff
@@ -170,6 +170,11 @@ function showConnectWindow(
     return { action: 'deny' };
   });
 
+  window.webContents.on('will-navigate', function (e, url) {
+    e.preventDefault();
+    void shell.openExternal(url);
+  });
```

Comment:
The less we have of this kind of stuff, the better, we already handle opening windows with `openExternal` in `setWindowOpenHandler`, seems like we are in control of the link there, so might be better to just add a blank target there

Reply:
I left a comment in COMPASS-5686, adding this handler just hides the issue that will also affect DE and I don't think hiding it (either by auto opening in external window or by logging) is a way to go here. If we want to add a listener, then it should be done in a way that will make sure we notice that we added a code that creates an issue

Reply:
I mean I understand the idea and I agree, but we still should have a different default. In my opinion, leaving this to "we should remember to add _target='blank'" is not enough, we need a bit more of safety net.

Reply:
We discussed f2f and decided to not automatically open external here and just prevent link from opening completely and debug log a warning that links should have a target blank to be able to open them through compass